web-marketplace/src/components/organisms/DashboardNavigation/DashboardNavigation.utils.tsx

```diff
@@ -0,0 +1,154 @@
+import { i18n } from '@lingui/core';
+import { msg } from '@lingui/macro';
+
+import { CogIcon } from 'web-components/src/components/icons/CogIcon';
+import CreditsIcon from 'web-components/src/components/icons/CreditsIcon';
+import { LogOutIcon } from 'web-components/src/components/icons/LogOutIcon';
+import { MembersIcon } from 'web-components/src/components/icons/MembersIcon';
+import { OrgProfileIcon } from 'web-components/src/components/icons/OrgProfileIcon';
+import { ProjectsIcon } from 'web-components/src/components/icons/ProjectsIcon';
+import { ShoppingBagIcon } from 'web-components/src/components/icons/ShoppingBagIcon';
+import { ShoppingCartIcon } from 'web-components/src/components/icons/ShoppingCartIcon';
+import { UserMenuIcon } from 'web-components/src/components/icons/UserMenuIcon';
+
+import { NOT_SUPPORTED_TOOLTIP_TEXT } from 'pages/Dashboard/MyProjects/MyProjects.constants';
+
+import { DashboardNavigationSection } from './DashboardNavigation.types';
+
+// Helper to wrap icons consistently
+const wrapIcon = (Icon: React.FC<any>, props: any = {}) => (
+  <span className="text-[24px] leading-none">
+    <Icon fontSize="inherit" linearGradient {...props} />
+  </span>
+);
+
+// Get credits section
+const getCreditsSection = (
+  loginDisabled: boolean,
+  collapsed: boolean,
+): DashboardNavigationSection => ({
+  // Direct translation for headings
+  heading: collapsed ? i18n._(msg`Credits`) : i18n._(msg`Manage credits`),
+  items: [
+    {
+      label: i18n._(msg`Portfolio`),
```

Comment:
let's avoid using `i18n._` in function that can be used within a useMemo, which is the case here, see https://lingui.dev/guides/lazy-translations#memoization-pitfall
let's rather pass `_: TranslatorType` as param (see existing examples in the code)